editoast/src/views/infra/errors.rs

```diff
@@ -65,10 +66,9 @@ enum ListErrorsErrors {
 }
 
 #[derive(QueryableByName, Debug, Clone, Serialize, Deserialize, PartialEq)]
-#[serde(deny_unknown_fields)]
 pub struct InfraError {
-    #[diesel(sql_type = Json)]
-    pub information: JsonValue,
+    #[diesel(sql_type = Jsonb)]
```

Comment:
You can be more precise here with DieselJson

Reply:
could please give a example?

Reply:
```suggestion
    #[diesel(sql_type = DieselJson<SchemaInfraError>)]
```

Reply:
Cf https://github.com/osrd-project/osrd/pull/6200#discussion_r1434901650, this PR might not be the best place to address this change.